test/qa.jl

```diff
@@ -0,0 +1,11 @@
+using MuladdMacro, Aqua
+@testset "Aqua" begin
+    Aqua.find_persistent_tasks_deps(MuladdMacro)
+    Aqua.test_ambiguities(MuladdMacro, recursive = false)
+    Aqua.test_deps_compat(MuladdMacro)
+    Aqua.test_piracies(MuladdMacro)
+    Aqua.test_project_extras(MuladdMacro)
+    Aqua.test_stale_deps(MuladdMacro)
+    Aqua.test_unbound_args(MuladdMacro)
+    Aqua.test_undefined_exports(MuladdMacro)
```

Comment:
I'd suggest using
```suggestion
    Aqua.test_all(MuladdMacro; test_ambiguities=(recursive = false,))
```
to not miss out on new Aqua tests in the future.

Reply:
For this package, that would likely be better.
However, for some of the more complicated SciML packages, it became too unwieldy to pass all the different keywords as named tuples.
I have the (minor) preference to apply the same template in all the SciML packages.
I'm committed to monitoring Aqua releases for new features.

Reply:
> I have the (minor) preference to apply the same template in all the SciML packages.

Is that possible if different packages need different keyword arguments? If they have to be different anyway, using `test_all` might reduce the maintenance burden for the simple ones at least.

Reply:
CI does not pass now.